examples/cfchannel/input_cfchannel.in

```diff
@@ -21,7 +21,10 @@ beam.mutpt = 0.0
 ###############################################################################
 # Beamline: lattice elements and segments
 ###############################################################################
-lattice.elements = constf1
+lattice.elements = monitor constf1 monitor
```

Comment:
Good point.

Currently, the file name for the data series created by a monitor object is taken from the name. If we want to different series, we can call them `monitor_begin` and `monitor_end`. (We can also make it custom, as in the Python example, with an additional sub-option.)

Technically, multiple monitors like this are not often needed, because within the data series of even a single monitor we can pick different steps along `s` from the outputs in the file. In the analysis scripts, you see that as `series.iteration[step]`.

I plan to add the meta-data of the reference particle in a future PR (instead of from an ASCII file only), which will make this even easier to select along the beamline from the output file alone.

Reply:
"Technically, multiple monitors like this are not often needed, because within the data series of even a single monitor we can pick different steps along s from the outputs in the file. In the analysis scripts, you see that as series.iteration[step]."

I'm not sure I follow, because I thought the purpose of each "monitor" was to store the particle data at a specific lattice location s.  I will try to understand the implemented monitor logic better before commenting further.